functions/src/index.ts

```diff
@@ -268,12 +268,22 @@ export async function handleUserStudyChangesImpl(
     `Couldn't find Glean Study ID for user ID ${userID} and Firebase study ID ${firebaseStudyID}. Aborting Glean ping process.`
   );
 
+  const schemaNamespace = await getSchemaNamespaceforStudy(studyID);
+
+  // Without the schemaNamespace, study-related Glean pings
+  // won't be tracked properly by the shredder in the future.
+  // This is bad and should be flagged for inspection
+  assert(
```

Comment:
Do you know if this will get logged the same as an exception, so we can get alerts etc?

Reply:
Yep, confirmed that these will show up in Error Reporting, e.g.:
<img width="1019" alt="Screen Shot 2022-05-23 at 2 12 24 PM" src="https://user-images.githubusercontent.com/7362472/169890094-a472c068-66b2-4949-8589-6f97f2cffb57.png">